packages/frontend/src/components/Bid/BumpBid/BumpBidForm.tsx

```diff
@@ -16,18 +16,28 @@ interface BumpBidProps {
   userBid: BidWithPlace
   newBid: BigNumber
   bumpAmount: BigNumber
+  minimumIncrement: BigNumber
   setBumpAmount: (val: BigNumber) => void
   setView: (state: TxFlowSteps) => void
   bids: BidWithPlace[]
 }
 
-export const BumpBidForm = ({ userBid, newBid, bumpAmount, setBumpAmount, setView, bids }: BumpBidProps) => {
+export const BumpBidForm = ({
+  userBid,
+  newBid,
+  bumpAmount,
+  minimumIncrement,
+  setBumpAmount,
+  setView,
+  bids,
+}: BumpBidProps) => {
   const { account } = useEthers()
   const userBalance = useEtherBalance(account)
-  const minimumIncrement = parseEther('0.01')
   const notEnoughBalance = userBalance !== undefined && bumpAmount.gt(userBalance)
   const bidTooLow = bumpAmount.lt(minimumIncrement)
 
+  useEffect(() => setBumpAmount(minimumIncrement), [minimumIncrement, setBumpAmount])
```

Comment:
All the dependencies of this `useEffect`  call are passed as a prop to this component, so they are available in `BumpBidFlow`  component. It makes more sense to put this call there.